sass/style.scss

```diff
@@ -0,0 +1,18 @@
+@import
+'fonts/Helvetica/helvetica';
```

Comment:
We can't use unlicensed fonts:
https://www.myfonts.com/fonts/linotype/neue-helvetica/licensing.html
https://www.myfonts.com/fonts/linotype/helvetica/licensing.html

Reply:
Agree that is something to be removed but I'd say the designing info should come much more accurate next time. That means... take into consideration these problems before implementing everything